src/node.h

```diff
@@ -228,7 +228,8 @@ class MultiIsolatePlatform;
 class InitializationResultImpl;
 
 namespace ProcessFlags {
-enum Flags : uint64_t {
+// uint32_t is lock-free
+enum Flags : uint32_t {
```

Comment:
```suggestion
// TODO: Switch to uint32_t to match init_process_flags in node.cc
enum Flags : uint64_t {
```

(unless you are okay with this PR being semver-major due to the ABI change – but even then the comment might be confusing without making it clear that it refers to std::atomic.)